### PR TITLE
Use minimist instead of gulp-util

### PR DESCRIPTION
**Pull request by shu-mutou**
Due to gulp-util was deprecated, use minimist instead.

### CODE REVIEW

File: aio/gulp/conf.js [hunk 1]
```diff
@@ -69,6 +69,11 @@ const version = {
  */
 const imageNameBase = 'kubernetes-dashboard';
 
+/**
+ * Arguments
+ */
+const argv = minimist(process.argv.slice(2));
```
Comment (maciaszczykm) [inline aio/gulp/conf.js]: What does this exactly mean? Why there is `.slice(2)`?

Comment (shu-mutou) [reply aio/gulp/conf.js]: `process.argv` has [0] as `node`, [1] as `gulp` and [2] as `serve`. So we need after them.

**Comment by shu-mutou** [comment]
I found `gulp-license-check` also uses `gulp-util`, and `minimist` used by lots of other packages. So, `package-lock.json` was not updated by `npm i`. 😨

File: aio/gulp/conf.js (hunk 1 above)
Comment (maciaszczykm) [reply aio/gulp/conf.js]: Makes sense, thank you.

**Comment by maciaszczykm** [comment]
Ah, that might be the case.

/lgtm
/approve

**Pull request merged**